src/Microsoft.OData.Core/Json/ODataUtf8JsonWriter.cs

```diff
@@ -47,6 +47,18 @@ internal sealed class ODataUtf8JsonWriter : IJsonWriter, IDisposable, IAsyncDisp
         private bool disposed;
         private const int chunkSize = 2048;
         /// <summary>
+        /// Content type of a value being written using TextWriter
+        /// </summary>
+        private string currentTextWriterContentType;
+        /// <summary>
+        /// Current stream for writing a binary property.
+        /// </summary>
+        private Stream binaryValueStream = null;
+        /// <summary>
+        /// Current textwriter for writing a stream property.
+        /// </summary>
+        private TextWriter textWriter = null;

```

Comment:
I think it'd be better to declare this field from partial class in _src/Microsoft.OData.Core/Json/ODataUtf8JsonWriter.TextWriter.cs_